src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.FileTimeToSystemTime.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32)]
+        internal static unsafe extern bool FileTimeToSystemTime(in long lpFileTime, out Interop.Kernel32.SYSTEMTIME lpSystemTime);
```

Comment:
Should all these APIs just take pointers and return BOOL to avoid marshalling overhead?

I have done looked at which ones are hot vs. which ones are not hot; but I would do it to all of them for consistency.

Reply:
I will do that. The definitions originally come from CoreRT and I kept them as-is, but avoiding marshaling overhead makes a lot of sense.